src/app/features/issue/providers/caldav/caldav-issue/caldav-issue.model.ts

```diff
@@ -0,0 +1,14 @@
+export type CaldavIssueReduced = Readonly<{
+  id: string;
+  completed: boolean;
+  item_url: string;
```

Comment:
This is not the same thing as issueUrl$ i suppose?

Reply:
Its the link to directly download the vcalendar entry. I considered to use it for updates first. But I think its not used anywhere at the moment.